docs/src/guides/laravel/deploy/octane.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Laravel Octane"
+sidebarTitle: "Octane"
+weight: -90
+description: Improve Laravel apps performance serving them with Laravel Octane.
+---
+
+[Laravel Octane](https://laravel.com/docs/octane) aim at improving Laravel applications performance by serving them using high-powered application servers, including [Swoole](https://github.com/swoole/swoole-src), [Open Swoole](https://openswoole.com/), and [RoadRunner](https://roadrunner.dev/).
+
+{{< note theme="warning" >}}
```

Comment:
```suggestion
{{< note >}}
```

I'd reserve warnings for situations where data might be lost or some other bad consequence could come from an action.